contrib/structure-test.sh.tpl

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -ex
```

Comment:
I generally try to avoid `-x` in what's checked in.

Reply:
fair